package.json

```diff
@@ -53,7 +53,7 @@
     "@vue/cli-plugin-e2e-nightwatch": "^3.9.2",
     "@vue/cli-plugin-eslint": "^3.8.0",
     "@vue/cli-plugin-unit-mocha": "^3.8.0",
-    "@vue/cli-service": "^3.8.0",
+    "@vue/cli-service": "^4.1.2",
```

Comment:
will bumping a major version affect anything?  wonder if we sync the rest of vue/cli-*

Reply:
Yeah thought about it.. gonna bump all things major in a new branch. I was scared.